oecophylla/distance/mash.rule

```diff
@@ -1,99 +1,51 @@
 rule mash_sketch:
     """
-    Sketches a trimmed and host-filtered fastq file.
-
-    There is almost no documentation for this tool, so it's problematic.
-
-    Relevant parameters might be:
-    -b : use bloom filtering on kmers to reduce impact of low-freq erros.
-    -m N: use explicit depth filtering on kmers (bigger memory impact than bloom)
+    Sketches a quality-controlled fastq file with mash.
     """
     input:
-        forward = qc_dir + "{sample}/filtered/{sample}.R1.trimmed.filtered.fastq.gz",
-        reverse = qc_dir + "{sample}/filtered/{sample}.R2.trimmed.filtered.fastq.gz"
+        forward = qc_dir + "{sample}/filtered/{sample}.R1.trimmed.filtered.fastq.gz"
     output:
-        sketch = distance_dir + '{sample}/mash/{sample}.msh'
+        distance_dir + '{sample}/mash/{sample}.msh'
     params:
-        mash = config['software']['mash'],
-        seqtk = config['software']['seqtk'],
-        mash_params = config['params']['mash']['other'],
-        depth = config['params']['mash']['depth']
+        env = config['envs']['distance'],
```

Comment:
trailing comma?

Reply:
no. it is required between lines or variables